uploader.json

```diff
@@ -483,6 +483,11 @@
             "pattern": [],
             "symbolic_name": "lvs_noheadings"
         },
+        {
+            "command": "/sbin/lvs --nameprefixes --noheadings --separator='|' -a -o lv_name,lv_size,lv_attr,mirror_log,vg_name,devices,region_size,data_percent,metadata_percent --config='global{locking_type=0} devices{filter=[\"a|.*|\"]}'",
```

Comment:
@lonicerae - the `lvs_noheadings_all` has been updated as `/sbin/lvs --nameprefixes --noheadings --separator='|' -a -o lv_name,lv_size,lv_attr,mirror_log,vg_name,devices,region_size,data_percent,metadata_percent,segtype --config='global{locking_type=0} devices{filter=[\"a|.*|\"]}'` by https://github.com/RedHatInsights/insights-core/pull/1331/files. please update it correspondingly.

Reply:
@xiangce @JoySnow  , could you please explain why we need to add segtype to the `_noheadings_all` parser?

Reply:
@lonicerae This 'segtype' info is required by some new plugins.

Reply:
@lonicerae - 
There is no any docstring notification to clarify the parser with `_all` and without `_all`, and I think that's why @JoySnow has added the `segtype` to both the `lvs_noheadings` and `lvs_noheandings_all` at the same time in https://github.com/RedHatInsights/insights-core/pull/1331

As you said in the meeting, after adding the `{filter=[\"a|.*|\"]}'`, the execution of this command will consume much more time than before adding it, especially there are many columns (`lv_name,lv_size,lv_attr,mirror_log,vg_name,devices,region_size,data_percent,metadata_percent,segtype`) need to be outputted.

To save the execution time of the command and shrink the collected data size, I would be better to remove all the unnecessary columns/options from the command.

And, since this is a newly added spec, and currently no rules depend on it, it's a good chance to update this command to a better size compare with the point that some rules depended on it.

Reply:
Let's drop this update at the moment until we have a better solution.